.github/workflows/test.yaml

```diff
@@ -39,4 +39,9 @@ jobs:
       - name: Install dependencies
         run: poetry install --no-dev --no-ansi --no-interaction
       - name: Run tests
+        env:
+          POSTGRES_DB_NAME: postgres
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB_HOST: localhost
```

Comment:
Do you also want to add POSTGRES_DB_PORT=5432 since we set it in our docker-compose.yml

Reply:
Sure, being explicit is a good idea